client/my-sites/stats/activity-log-tasklist/index.jsx

```diff
@@ -105,7 +108,11 @@ class ActivityLogTasklist extends Component {
 			items = [ item.slug ];
 			trackDismiss( item );
 		} else {
-			items = union( plugins.map( p => p.slug ), themes.map( p => p.slug ) );
+			items = union(
+				coreWithUpdate.map( p => p.slug ),
```

Comment:
what does p stand for? Can we spell it out?

Reply:
The slug would be `wordpress`. I kept it this way because it's the same. Will be more verbose about it.